common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -381,16 +404,42 @@ export class SendTransaction extends React.Component {
       unit
     });
   };
+
+  generateTx = async () => {
+    const { nodeLib, wallet } = this.props;
+    const address = await wallet.getAddress();
+
+    try {
+      const transaction = await nodeLib.generateTransaction(
+        {
+          to: this.state.to,
+          from: address,
+          value: this.state.value,
+          gasLimit: this.state.gasLimit,
+          gasPrice: this.props.gasPrice,
+          data: this.state.data,
+          chainId: this.props.network.chainId
+        },
+        wallet
+      );
```

Comment:
Checks

**to: valid address**
https://github.com/ethereumjs/ethereumjs-util/blob/cfeac0fc49c00111a77124bbe76011e4b1129cf9/index.js#L432-L434

**from: valid address**

gasLimit: greater than 21000, less than 5M (the network has an ever-shifting limit). integer.

gasPrice: this should be in wei. warn user if over 250 GWEI (250000000000 wei). Do not send if over 1000(1000000000000) GWEI (tell them to contact us to explain why they would want to send that expensively?
) 

https://www.myetherwallet.com/helpers.html is a good place to convert